tests/integration/test_payment_initiation.py

```diff
@@ -20,22 +22,29 @@ def payments_after_recipient_creation(client, recipient_id):
     assert payment_id is not None
     assert response['status'] is not None
 
-    # create payment token
-    response = client.PaymentInitiation.create_payment_token(
-        payment_id,
-    )
-    assert response['payment_token'] is not None
-    assert response['payment_token_expiration_time'] is not None
+    # create link token
+    response = client.LinkToken.create({
```

Comment:
is there another test for `client.LinkToken.create` somewhere else already?

Reply:
Yep! There should already be a test for `client.LinkToken.create`